src/utils/DeviceUtils/DeviceUtils.ts

```diff
@@ -0,0 +1,25 @@
+import {Dimensions, NativeModules, Platform} from 'react-native';
+import {PlatformType} from './DeviceUtils.types';
+
+const isAndroid = Platform.OS === 'android';
+const isIos = Platform.OS === 'ios';
+const isPlatform = (T: PlatformType) => Platform.OS === T;
+
+const locale = isIos
+  ? NativeModules.SettingsManager.settings.AppleLocale
+  : NativeModules.I18nManager.localeIdentifier;
+
+const getDeviceWidth = (dim: 'window' | 'screen' = 'window') =>
```

Comment:
Check if this changes between landscape and portrait. If not then make this as variable

Reply:
Yes, It is changing on device screen changing from portrait to landscape the horizontal side length is being taken as width.